packages/suite/src/views/suite/bridge/index.tsx

```diff
@@ -79,7 +79,7 @@ const BridgeDesktopNote = styled(P)`
 `;
 
 const StyledImage = styled(Image)`
-    @media screen and (max-height: ${variables.SCREEN_SIZE.MD}) {
+    @media screen and (max-height: ${variables.SCREEN_SIZE.LG}) {
```

Comment:
No images for the poor! 😁  This should seriously be handled better on the design side. Modals should be designed so they fit to most of screens (768px height is a still good rule of thumb). Assuming everyone has at least full hd display (with suite almost on fullscreen) is wrong, but 🤷‍♂️. (Rant is aimed for a product/design team, not you. I just couldn't help myself).